msteams-platform/concepts/messaging-extensions/create-extensions.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Action based messaging extensions
+description: Describes how to develop action based messaging extensions
+keywords: teams messaging extensions messaging extensions search
+ms.date: 11/07/18
+---
+
+# Initiate actions with messaging extensions (Developer preview)
+
+Action based messaging extensions allow your users to trigger actions in 3rd party services while inside of Teams.
```

Comment:
Shouldn't it be "action-based"?

Reply:
Also we avoid using the term "third party" - I *hate* that term. say "external services"

Reply:
Changed to action-based. I agree.

Third party removed.